src/tls_wrap.cc

```diff
@@ -930,6 +930,16 @@ void TLSWrap::EnableKeylogCallback(
 # define HAVE_SSL_TRACE 0
 #else
 # define HAVE_SSL_TRACE 1
+static void msg_callback(
```

Comment:
Style: the name should be something like `TraceMessageCallback`. Just `msg_callback` is arguably also a little too generic.